cluster/manifests/kube-dns/depl-kube-dns.yaml

```diff
@@ -27,6 +27,18 @@ spec:
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                  - key: application
+                    operator: In
+                    values:
+                    - kube-dns-backup
```

Comment:
This needs to be changed to kube-dns-b as well.

Reply:
Fixed: https://github.com/zalando-incubator/kubernetes-on-aws/pull/812